lua/tabline.lua

```diff
@@ -863,31 +863,31 @@ function M.setup(opts)
     end
   end
 
-  if opts.options.always_show_tabs or opts.options.show_tabs_always then
+  if (opts.options.always_show_tabs or opts.options.show_tabs_always) ~= nil then
```

Comment:
I just tested the following in the lua REPL.

```lua
print(nil or false) -- false
print(false or nil) -- nil
```

Maybe we should check each option individually instead?

Reply:
So the logic should be "if one of them is not `nil` then set the option to the non-nil one else goes default"?
Sounds like a lot of `if`s, but I guess you are right.

Reply:
I like the suggestion in your edit.